src/benchmarks/micro/libraries/System.Formats.Tar/Perf.TarFile.cs

```diff
@@ -67,15 +67,15 @@ public async Task CreateFromDirectory_Path_Async()
         [Benchmark]
         public void CreateFromDirectory_Stream()
         {
-            using MemoryStream ms = new MemoryStream();
+            MemoryStream ms = new MemoryStream();
```

Comment:
these benchmarks were correct, why would you like to change them?

```suggestion
            using MemoryStream ms = new MemoryStream();
```